### ISSUE TITLE
pyright does not know about `AST.lineno`

### ISSUE THREAD

**Issue by crhf** [microsoft/pyright#8490]
**Describe the bug**
pyright doesn't know about `AST.lineno`.

**Code or Screenshots**
Here is `a.py`:

```python
import ast

def foo(tree: ast.AST):
   print(ast.lineno) 
```

Running `pyright a.py` from command line gives:

```
/path/to/a.py
  /path/to/a.py:4:16 - error: Cannot access attribute "lineno" for class "AST"
    Attribute "lineno" is unknown (reportAttributeAccessIssue)
1 error, 0 warnings, 0 informations
```

I am using `Python 3.11.7` and `pyright 1.1.372`.

**Comment by erictraut** [comment]
Pyright's "knowledge" of symbols is limited to what is provided by library authors and stub providers. In this case, the type information for the `ast` module comes from the community-maintained [typeshed project](https://github.com/python/typeshed/tree/main). If you think there's a missing symbol in the [ast.pyi](https://github.com/python/typeshed/blob/main/stdlib/ast.pyi) stub, please file a bug report or a PR in that project.
